### PR TITLE
Upgrade to webpack 5

### PR DESCRIPTION
**Pull request by marcbachmann**
### 💥 BREAKING CHANGE
The changes here are only compatible with webpack 5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,9 +34,9 @@
   },
   "dependencies": {
     "fastify-plugin": "^3.0.0",
-    "middie": "^5.1.0",
-    "webpack": "^4.43.0",
-    "webpack-dev-middleware": "^3.7.0",
+    "middie": "^5.3.0",
+    "webpack": "^5.45.1",
```
Comment (marcbachmann) [inline package.json:38]: Maybe we should move those webpack dependencies into `peerDependencies`

**Comment by marcbachmann** [comment]
I guess I'll need to publish that myself.

**Comment by IanVS** [comment]
@marcbachmann did you publish a fork of this project?  I'm looking for something I can use with webpack 5 and fastify 4.

**Comment by marcbachmann** [comment]
https://github.com/livingdocsIO/fastify-webpack-hmr
https://www.npmjs.com/package/@livingdocs/fastify-webpack-hmr
Haven't tested it yet with fastify 4, but I'm using it in here with fastify 3 since a longer time: https://github.com/livingdocsIO/fastify-webpack

**Pull request closed without merging**